rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -247,6 +247,11 @@ def verify_event_record(
             raise EventRecordDataError("event record not generated! File is empty")
 
     # verify event record for a particular event type
+    ceph_version_id, _ = utils.get_ceph_version()
+    if float(ceph_version_id[1]) >= 9:
```

Comment:
@ckulal 
this may fail when we have v20.2.0,
we can take it later then..

Reply:
Done.
pass log: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/fix_noncurrent/noncurrent-pass-new